test/distributed/org/apache/cassandra/fuzz/harry/examples/RangeTombstoneBurnTest.java

```diff
@@ -84,41 +84,35 @@ public void rangeTombstoneBurnTest() throws Throwable
 
                 DataTracker tracker = new DefaultDataTracker();
                 modelChecker.init(new ReplayingHistoryBuilder(seed, maxPartitionSize, STEPS_PER_ITERATION, new DefaultDataTracker(), sut, schema, rf, SystemUnderTest.ConsistencyLevel.ALL))
-                            .step((history) -> {
-                                      int rowIdx = entropySource.nextInt(maxPartitionSize);
-                                      int partitionIdx = partitions[entropySource.nextInt(partitions.length)];
+                            .step((history, rng) -> {
```

Comment:
It doesn't look like it matters, but should the other calls to `step()` use the `rng` passed by `ModelChecker#run()`? Or to flip that around, does this first step even need to?

Reply:
well spotted!